### PR TITLE
Daily uploads

### PR DESCRIPTION
**Pull request by MunteanuM**
Add possibility to import Movies

### CODE REVIEW

File: basepage/views.py [hunk 1]
```diff
@@ -35,6 +42,92 @@ def contact(response):
 def films(response):
     return render(response, 'basepage/films.html', {})
 
+@ratelimit(key='ip')
+def playing_films(response):
+    return render(response, 'basepage/playing_films.html', {})
+
+def import_csv(request):
+    file = r'C:\Users\mihamunteanu\DjangoProjects\cinema\imdb_top_1000.csv'
```
Comment (maxinasa) [inline basepage/views.py]: This is definitely not ok with the hardcoded path.

File: app_users/forms.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    class Meta:
+        model = User
+        fields = [
+            'username',
+            'first_name',
+            'last_name',
+            'email',
+            'password1',
+            'password2',
+            ]
```
Comment (maxinasa) [inline app_users/forms.py:20]: I'm not exactly sure, but I feel like a lot of these changes have already been present in a different PR.

File: app_users/templates/authentication/login.html [hunk 3]
```diff
@@ -20,6 +20,6 @@
     <button type="submit" class="btn btn-primary">Log In</button>
 </form>
 
-<a href="http://127.0.0.1:8000/login/">Sign up</a>
```
Comment (drejleeo) [inline app_users/templates/authentication/login.html:23]: Don't reference the entire url. Use relative paths

**Review by drejleeo — CHANGES_REQUESTED** [review]
Add `.pyc` files to `.gitignore` and delete them, otherwise they'll remain on git.

**Pull request closed without merging**